vmck/backends/submission.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+
+
+def task(job, options):
+    return {
+        'name': 'submission-handler',
+        'driver': 'docker',
+        'config': {
+            'image': f'vmck/vagrant-vmck:{options["env"]["vagrant_tag"]}',
+            'command': '/src/submission/launch.sh',
+        },
+        'env': {
+            'ARCHIVE_URL': options['env']['archive'],
+            'SCRIPT_URL': options['env']['script'],
+            'VMCK_URL': settings.VMCK_URL,
+            'INTERFACE_ADDRESS': options['env']['interface_address'],
```

Comment:
What does `interface_address` do? I found https://github.com/vmck/vagrant-vmck/blob/e59aafa7d8f0c73cdfc2f85b2b27fe197b833b39/submission/launch.sh#L18 but the name is not helpful. And this, along with `submission_id`, means adding stuff specific to `acs-interface` in `vmck` which we should really avoid.

How about making this a bit more generic? Define a variable `VMCK_CALLBACK_URL` which, if set, should be called from vagrant-vmck when it's done. Include the `/done/` part in that value, and `SUBMISSION_ID` too. The full URL could be `$acs_interface_url/submission/$submission_id/done` (and no trailing slash pls).